### ISSUE TITLE
Tab order on create profile page needs some attention

### ISSUE THREAD

**Issue by EmmaDawsonDev** [danielcranney/profileme.dev#27]
Some of the things from the sidebar are still tabbable even when the sidebar is not visible making the tab order seem strange. You can see in the screen shot that after tab stop 3 the line disappears to the left then comes back to 9 which means there are 5 tabstops which are not visible and may cause confusion for keyboard users

<img width="948" alt="image" src="https://user-images.githubusercontent.com/57045550/175105676-26751da3-b7e7-4d79-bf77-702d21bbe47e.png">
